I don't know about you, but the fact that an extra action
is required to clear the now playing queue annoys me. Massively.

This patch disables this 'do you want to clear' prompt on the now
playing queue and just immediately clears it. I'm pretty sure if
I right click 'Clear' I mean it. Bad design.

--- gmpc-11.8.16/src/browsers/playlist3-current-playlist-browser.c	2011-08-16 23:09:55.000000000 -0400
+++ gmpc-11.8.16/src/browsers/playlist3-current-playlist-browser.c	2016-01-13 12:17:28.790894381 -0500
@@ -1058,19 +1058,7 @@
 }
 static void pl3_current_playlist_browser_clear_playlist(void)
 {
-    GtkWidget *delete;
-
-    delete = gtk_button_new_from_stock(GTK_STOCK_CLEAR);
-    g_signal_connect(G_OBJECT(delete),
-            "clicked",
-            G_CALLBACK(pl3_current_playlist_browser_clear_playlist_real),
-            NULL);
-
-    /* show message */
-    playlist3_message_show(pl3_messages,
-            _("Are you sure you want to clear the play queue?")
-            ,USER_FEEDBACK);
-    playlist3_message_add_widget(pl3_messages, delete);
+    pl3_current_playlist_browser_clear_playlist_real(); // Look, guys. I'm not going to hit 'ok' every time I clear my fricken music. Sorry.
 }
 
 static void pl3_current_playlist_browser_shuffle_playlist(void)
